README_EXEC-TTYNVT.md

```diff
@@ -0,0 +1,9 @@
+# exec-ttynvt
+
+The ttynvt executer is used to observe _ttynvt._tcp mdns services of io4edge devices. It starts a new ttynvt instance when a new service shows up and terminates the corresponding instance again when the service disappears.
```

Comment:
Please document how the /dev/xxx devices are named. ideally, the should be named like the /dev/tty<instance>